core/src/main/java/org/infinispan/commands/write/EvictCommand.java

```diff
@@ -38,11 +38,17 @@ public Object perform(InvocationContext ctx) throws Throwable {
    }
 
    @Override
-   public void notify(InvocationContext ctx, Object value, Metadata previousMetadata) {
-      /**
-       * We don't notify in pre for evictions - the notification is done in
-       * {@link org.infinispan.interceptors.locking.ClusteringDependentLogic.AbstractClusteringDependentLogic#notifyCommitEntry(boolean, boolean, boolean, org.infinispan.container.entries.CacheEntry, org.infinispan.context.InvocationContext, org.infinispan.commands.FlagAffectedCommand, org.infinispan.container.entries.InternalCacheEntry)}
-        */
+   public void notify(InvocationContext ctx, Object value, Metadata previousMetadata, 
+         boolean isPre) {
+      // Eviction has no notion of pre/post event since 4.2.0.ALPHA4.
```

Comment:
I think this comment should be added to the CacheEntryInvalidatedEvent so that it gets included in javadoc